nodebb-theme-fdk/languages/en-GB/fdk.json

```
@@ -0,0 +1,39 @@
+{
+  "welcome": {
+    "header": "Community",
+    "ingress": "Welcome to the Felles datakatalog community! We're happy to see you here.",
+    "paragraph": "Here, you can ask questions about anything Felles datakatalog and information management related, participate in discussions, showcase your work, make new connections, and discover new projects for development and innovation. The community is open, but you must register a user before writing posts and creating new topics. The purpose of this community is to facilitate value creation through data for the whole of society - join and contribute to increasing knowledge, openness, and innovation."
```

```suggestion
    "paragraph": "Here, you can ask questions about anything related to Felles datakatalog and information management, participate in discussions, showcase your work, make new connections, and discover new projects for development and innovation. The community is open, but you must register a user before writing posts and creating new topics. The purpose of this community is to facilitate value creation through data for the whole of society - join and contribute to increasing knowledge, openness, and innovation."
```
Ikke nødvendigvis mer riktig, men jeg følte det ga setningen litt bedre flyt